context/src/main/java/io/grpc/Context.java

```diff
@@ -657,8 +658,35 @@ private Object lookup(Key<?> key) {
    * cancelled and which will propagate cancellation to its descendants. To avoid leaking memory,
    * every CancellableContext must have a defined lifetime, after which it is guaranteed to be
    * cancelled.
+   *
+   * <p>This class must be cleaned up by either calling {@link #close} or {@link #cancel}.
+   * {@link #close} is equivalent to calling {@code cancel(null)}. It is safe to call the methods
+   * more than once, but only the first call will have any effect.
+   *
+   * <p>Blocking code can use the try-with-resources idiom:
+   * <pre>
```

Comment:
Get rid of the extra newlines. Take a look at the generated documentation to see how it looks.